mitodl/open-discussions
# Elasticsearch 7.10 upgrade

#### Pre-Flight checklist


#### What are the relevant tickets?
updates https://github.com/mitodl/open-discussions/issues/3494

#### What's this PR do?
This PR upgrades from ES 6.8 to ES 7.10 in preparation for the OpenSearch swap.

#### How should this be manually tested?
Pull the branch locally and run `docker build` and `docker up` to run the new version of Elasticsearch. 

I would then:
- run tests
- check that you can use the search API manually using postman or fetch or some tool of your choice - specifically as OCW might make requests
- make sure that pages within Open that are using the search API function (new open demo, infinite, learn, open discussions)

#### Where should the reviewer start?
Pull the branch and play with it, honestly. I can run all the tests in the world and I can try to break it, but ultimately it's best if we have many people poke it from all sides.

#### What GIF best describes this PR or how it makes you feel?
![ImprobableFarflungHapuka-max-1mb](https://user-images.githubusercontent.com/7756053/236240398-50900256-d6bd-4196-8694-c60a227b506b.gif)
